### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by OlaShabalina**
## Overview of the Pull Request:
- Update Readme;
- Add links to github project stats images;

Feel free to share your thoughts on what else can be added or propose changes.

## link to Trello card or Github issue
Trello card: https://trello.com/c/a5FGjdnI
Github issue: https://github.com/CodeforAustralia/council-emissions-calculator/issues/221

## How Has This Been Tested?
- Visually;
- Grammar was tested in MS Word;
- Links have to be clicked to check they are pointing to the right direction.

## Pull Request Checklist:

Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!

- [x] Have you included a link Trello card / Github issue and written sufficient description to help others review your work?
- [x] Is your pull request up-to-date with `main` branch?
- [x] Have you checked That code is well formatted? Did `npx prettier --check .` return no warnings/errors?
- [ ] Have you written instructions on how to test that your changes work as expected?
- [x] Have you tested your work thoroughly on your local machine to ensure you are not breaking anything?

### CODE REVIEW

**Comment by OlaShabalina** [comment]
[[DEV]: Update Readme to reflect the current stage of the project and new contribution model](https://trello.com/c/a5FGjdnI/20-dev-update-readme-to-reflect-the-current-stage-of-the-project-and-new-contribution-model)

File: README.md
Comment (Amutha37) [inline README.md]: Great work Olga. Just a small additions .
Line 5
'Civic Maker's Council's Carbon Emissions Calculator Project'

File: README.md
Comment (Amutha37) [inline README.md]: Awesome work ! The word carbon need to be added.
Council carbon emission calculator system diagram.

File: README.md
Comment (Amutha37) [inline README.md]: Should we use the word 'website' or  `to view the carbon emission survey app'. Which related to what actually they will view as the go into locallhost:3000.

File: README.md
Comment (Amutha37) [inline README.md:100]: `# Data Dependencies`  will be more appropriate due to we have used the word before for `npm dependencies.`

File: README.md
Comment (Amutha37) [inline README.md:52]: We can use the word  `npm dependencies`  to differentiate with `data dependencies` below this lines of notes. 
So this will be : 
`Install npm dependencies using your preferred package manager:`

**Review by Amutha37 — DISMISSED** [review]
Great documentation with small changes needed.

File: README.md
Comment (OlaShabalina) [reply README.md]: changed

File: README.md
Comment (OlaShabalina) [reply README.md]: Not critical, it's an alt text, but changed anyway

File: README.md
Comment (OlaShabalina) [reply README.md]: Changed

File: README.md
Comment (OlaShabalina) [reply README.md:100]: Don't agree, those are not data dependencies. No repetition here, those are different sections.

Comment (Amutha37) [reply README.md:100]: All good

File: README.md
Comment (Amutha37) [reply README.md:52]: All go no need to change than.

File: README.md
Comment (OlaShabalina) [reply README.md:100]: will need approval then, can't proceed without it :grin:

File: README.md
Comment (hqtan) [reply README.md]: I disagree with the grammar of this phrase:
> Civic Maker's Council's Carbon Emissions Calculator project

I think this looks more correct:
** Civic Maker's Council Carbon Emissions Calculator project **

File: README.md
Comment (hqtan) [inline README.md]: a small suggestion: add link to slack channel

```suggestion
| Slack | [\#civic-makers-climatechange](https://codeforaustralia.slack.com/archives/C01CXCQPF8V) |
```

File: README.md
Comment (hqtan) [inline README.md]: I **love** your systems diagram @OlaShabalina !!! 😍 

A couple of suggestions for improvement I offer for your thoughts:
* the survey results are stored and processed in *Google Sheets*, not *Google Pages* 😅 
* Did you create your diagram with DrawIO (or Excalidraw)? Are you able to put a copy of the DrawIO file *and* the PNG image file in the repo? I would love to have your diagrams version controlled as well! 🙇🏽

File: README.md
Comment (hqtan) [inline README.md]: > 1. Create a new branch locally. _No need to fork the project, you can create a branch in the original one._

This statement is true, but it only applies to people who are listed as members in this repo. New contributors, who are not yet members of the repo, **will need to fork** this repo and then contribute their changes via the fork.

How about this:

```suggestion
Please refer to [this doc](https://docs.github.com/en/get-started/quickstart/fork-a-repo) for instructions on how to fork this repo and contributing your changes.

**N.B:** Please note well that the *default branch* for this repo is the **staging** branch, not the *main* branch; Any new pull request raised for this repo will have its changes pushed to **staging** branch by default when the PR is approved and merged.

The **staging** branch is hosted on our [Staging endpoint](http://council-emissions-calculator-spike-flax.vercel.app/). 

The **main** branch is hosted on our [Production endpoint](https://seechange.codeforaustralia.org/).

```

**Review by hqtan — CHANGES_REQUESTED** [review]
Hi @OlaShabalina ,

Thx a bunch for your PR 😍 

Your changes to the readme is a **definite** improvement to the current readme 💯 

I've left some suggestions for improvement for your consideration 🙇🏽

File: README.md
Comment (OlaShabalina) [reply README.md]: checked with chatGPT, both are correct @hqtan :joy: preference?

File: README.md
Comment (OlaShabalina) [reply README.md]: Loaded the file and updated "Google Sheets" reference

File: README.md
Comment (OlaShabalina) [reply README.md]: Either way, changed to the one recommended by @hqtan if it's ok with you @Amutha37

**Review by hqtan — APPROVED** [review]
lgtm! 🚀

File: README.md
Comment (hqtan) [reply README.md]: Hi @OlaShabalina ,

This one has been left open for a while... 
Let's go with my preference for the wording ok?

Comment (OlaShabalina) [reply README.md]: The problem is that @Amutha37  requested changes earlier and she would have to approve the PR for me to be able to unblock the merge.

**Pull request merged**
